### ISSUE TITLE
Questbook not detecting item v0.0.7

### ISSUE THREAD

**Issue by S0llux** [FireBall1725/LeftBehind-Modpack#22]
The clay quest doesn't detect the unfired clay bucket if it is crafted, it only identifies the spawned in version of the item.

**Comment by FireBall1725** [comment]
yep, HQM sucks, i want to move to something else, what I will probably end up doing is write the quests in HQM and then see if i can move them to something else.
